npm/icons/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@cypress/icons",
```

Comment:
are we going to continue publishing this NPM package? does anyone else use this NPM package? if we're only using it in the monorepo we could move this to `packages` and stop having to publish this to NPM

Reply:
We could stop, I did not think answering this question and archiving the repo + npm package would be part of this PR.